src/plugins/kibana_react/public/code_editor/editor_theme.ts

```diff
@@ -13,7 +26,13 @@ import chrome from 'ui/chrome';
 // NOTE: For talk around where this theme information will ultimately live,
 // please see this discuss issue: https://github.com/elastic/kibana/issues/43814
 
-const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+// Disable dark theme if unable to query client UI settings
+let IS_DARK_THEME = false;
+try {
+  IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
```

Comment:
Importing from `src/legacy` in `src/plugins` is not allowed.

You could use `useUiSetting` hook from `kibana_react` or pass the `theme:darkMode` setting from the New Platform yourself using props or React context.

Alternatively, in your case I see `themeName` is used only in one place.

![image](https://user-images.githubusercontent.com/9773803/69463283-7dbe8d80-0d7b-11ea-90cf-8ef5aa25249c.png)

Maybe you could just use one color in both modes, and remove the `themeName` variable.

Reply:
Yeah the plan is to use the `useUISetting` hook. That change is actually present here: https://github.com/elastic/kibana/pull/49358/files#diff-87b533494b88e6460945f12305da86b4R11